pyproject.toml

```diff
@@ -4,7 +4,9 @@ exclude = [
   "tools/cpplint.py",
   "tools/gyp",
   "tools/inspector_protocol",
-  "tools/eslint/node_modules",
+  "tools/eslint",
+  "tools/icu",
```

Comment:
Not sure about this one. I think the files in the `tools/icu` folder are nodejs-specific.